src/test/java/de/numcodex/feasibility_gui_backend/query/broker/direct/DirectBrokerClientCqlIT.java

```diff
@@ -0,0 +1,105 @@
+package de.numcodex.feasibility_gui_backend.query.broker.direct;
+
+import static de.numcodex.feasibility_gui_backend.query.QueryMediaType.CQL;
+import static de.numcodex.feasibility_gui_backend.query.collect.QueryStatus.COMPLETED;
+import static java.lang.String.format;
+import static java.util.Map.entry;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.timeout;
+import static org.mockito.Mockito.verify;
+
+import ca.uhn.fhir.context.FhirContext;
+import ca.uhn.fhir.rest.client.api.IGenericClient;
+import de.numcodex.feasibility_gui_backend.query.broker.QueryNotFoundException;
+import de.numcodex.feasibility_gui_backend.query.broker.SiteNotFoundException;
+import de.numcodex.feasibility_gui_backend.query.collect.QueryStatusListener;
+import de.numcodex.feasibility_gui_backend.query.collect.QueryStatusUpdate;
+import java.io.IOException;
+import java.util.Map;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+import org.junit.jupiter.api.TestInstance.Lifecycle;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.web.reactive.function.client.WebClient;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.containers.wait.strategy.Wait;
+import org.testcontainers.images.PullPolicy;
+import org.testcontainers.junit.jupiter.Testcontainers;
+import org.testcontainers.utility.DockerImageName;
+
+@Testcontainers
+@TestInstance(Lifecycle.PER_CLASS)
+@ExtendWith(MockitoExtension.class)
+@SuppressWarnings("NewClassNamingConvention")
+class DirectBrokerClientCqlIT {
+
+    private static final int ASYNC_TIMEOUT_WAIT_MS = 2000;
+    private static final Long TEST_BACKEND_QUERY_ID = 1L;
+
+    private final Map<String, String> CODE_SYSTEM_ALIASES = Map.ofEntries(
```

Comment:
This is used.

Reply:
I assume the "not" was missing from the comment...removed the map